tests/wpunit/Admin/NoticeTest.php

```diff
@@ -19,7 +19,7 @@ public function test_it_should_display_notice() {
 		$notice = new Notice();
 		$notice->add_notice( Notice::EXPIRED_KEY, 'uplink' );
 
-		$this->expectOutputString( '<div class="notice notice-warning"><p>Your license is expired<a href="https://evnt.is/195y" target="_blank" class="button button-primary">Renew Your License Now<span class="screen-reader-text"> (opens in a new window)</span></a></p>    </div>' );
+		$this->expectOutputString( '<div class="notice notice-warning"><p>Your license is expired</p></div>' );
```

Comment:
Shouldn't this still output the original markup unless you've filtered it?

Like, won't this break for everyone else?

Reply:
It's a bug that this outputs with a TEC link as the default. It really shouldn't and shouldn't require a custom filter in every plugin just to fix this. I'm only leaving the link for backwards compatibility with TEC plugins using uplink.

Reply:
I'll get another test in tomorrow

Reply:
Gotcha, I understand now. Yeah, I think one more test and this should be good to go 🚀